cypress/e2e/pages.cy.ts

```diff
@@ -22,3 +43,21 @@ describe("Screenshot pages", () => {
     });
   }
 });
+
+describe("Screenshot pages dark mode", () => {
```

Comment:
We should switch to Playwright and run a test suite with dark mode directly in system.

I think we should switch with Playwright because it's the best tool to use with Argos.

Reply:
I agree. I'll do it today